geode-core/src/main/java/org/apache/geode/internal/monitoring/ThreadsMonitoringImplDummy.java

```diff
@@ -32,9 +32,17 @@ public boolean startMonitor(Mode mode) {
   @Override
   public void endMonitor() {}
 
+  private static class DummyAbstractExecutor extends AbstractExecutor {
+    private static final DummyAbstractExecutor SINGLETON = new DummyAbstractExecutor();
+
+    private DummyAbstractExecutor() {
+      super("dummyExecutorGroup", 0L);
```

Comment:
From looking at other group names they are typically capitalized like class names and don't end in the word `Group`. So I recommend:

`"DummyExecutor"`

Reply:
done